src/libman/Contracts/HostInteraction.cs

```diff
@@ -97,11 +97,14 @@ public Task<Stream> ReadFileAsync(string relativeFilePath, CancellationToken can
             return FileHelpers.ReadFileAsStreamAsync(relativeFilePath, cancellationToken);
         }
 
-        public async Task<bool> CopyFile(string sourcePath, string destinationPath, CancellationToken cancellationToken)
+        public Task<bool> CopyFile(string sourcePath, string destinationPath, CancellationToken cancellationToken)
```

Comment:
should this be CopyFileAsync as it returns a task?

Reply:
It should be, but that would require changing the interface, and I don't want to make that change in this PR.

Reply:
OK, there's no public surface area concerns then, right? Should we log a separate issue to track that?

Reply:
At the moment, I don't believe any of the public surface area in this project are being used elsewhere.  This is something that we should probably clean up when/if we get to establishing an extensibility story.

Reply:
OK, will resolve. Seems easy enough to just change it now, but no big deal.